### PR TITLE
Adding documentation on switching loadbalancer credentials.

### PR DESCRIPTION
**Pull request by holingpoon**

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
I see your confusion. @holingpoon I think you should explain a bit more about why we are doing it and how to use it to switch context when _manually_ deploying a branch.

**Comment by holingpoon** [comment]
Will do. Adding more explanation now.

File: readme.md [hunk 1]
```diff
@@ -163,6 +163,30 @@ Once the application has been created via `eb create ...` all subsequent updates
 eb deploy ${environment name} --profile ${your AWS profile}
 ```
 
+#### Switching Loadbalancer settings between accounts
+
+Loadbalancer settings is required for HTTPS to run behind the load balancers for Elastic Beanstalk. Each Loadbalancer setting is unique to each AWS profile, so switching configuration files is required to deploy to different AWS profiles.
```
Comment (gkallenberg) [inline readme.md]: ...settings are required...

...is required when deploying to different...

File: readme.md [hunk 2]
```diff
@@ -163,6 +163,30 @@ Once the application has been created via `eb create ...` all subsequent updates
...
+
+| AWS Profile | Configuration File | 
+|---|---|
+| `nypl-digital-dev` | `https-nypl-digital-dev.config` | 
+| `nypl-sandbox` | `https-nypl-sandbox.config` | 
+
+When `development` is ready to merge into `qa`, only include `https-nypl-digital-dev.config` within `.ebextensions` directory. Since CI/CD is being performed via Travis CI, the following needs to be done on the local `qa` branch, after a merge with the most up-to-date local `development` branch, before pushing into remote `qa`:
```
Comment (gkallenberg) [inline readme.md]: ...within the `.ebextensions` directory.

...Travis CI, run the following commands on the local `qa` branch after a merge...

**Comment by holingpoon** [comment]
@EdwinGuzman I agree this is not ideal, I've requested bandwidth to look into CloudFormation templates to see if we can automate switching the credentials.

File: readme.md [hunk 3]
```diff
@@ -163,6 +163,30 @@ Once the application has been created via `eb create ...` all subsequent updates
...
+```bash
+git checkout qa
+git merge development
+git rm .ebextensions/https-nypl-sandbox.config
+git add .ebextensions/https-nypl-digital-dev.config
+git commit
+git push origin qa
+```
+
+At `git push origin qa`, Travis CI will be triggered and will deploy to QA server with the correct credentials.
```
Comment (gkallenberg) [inline readme.md]: Travis CI will deploy to the QA server with the correct credentials.

Shouldn't we mention production as well in this section regarding the appropriate credentials being deployed to the correct environment?

File: readme.md (hunk 1 above)
Comment (ktp242) [reply readme.md]: Much more clear now!!!! Now I know why we need this.

File: readme.md (hunk 3 above)
Comment (ktp242) [reply readme.md]: I think we should.

**Comment by ktp242** [comment]
Looks good! Just some fixes that Greg mentioned.

**Comment by holingpoon** [comment]
@gkallenberg Instructions also covers production, I will clarify in the instructions that the switching step is not required for merging `qa` into `master`/`production` branch since we are already using the credentials from `nypl-digital-dev` account.

File: readme.md [hunk 4]
```diff
@@ -163,6 +163,32 @@ Once the application has been created via `eb create ...` all subsequent updates
...
+```bash
+git checkout qa
+git merge development
+git rm .ebextensions/https-nypl-sandbox.config
+git add .ebextensions/https-nypl-digital-dev.config
+git commit
+git push origin qa
+```
+
+At `git push origin qa`, Travis CI will deploy to the QA server with the correct credentials..
```
Comment (gkallenberg) [inline readme.md]: Double period here.

File: readme.md (hunk 2 above)
Comment (gkallenberg) [reply readme.md]: You didn't like these suggestions? ^^

Comment (holingpoon) [reply readme.md]: More like missing it as I scroll by. Let me go to GitHub "Files Changed" section to go through one more time :smile:

**Pull request merged**